src/include/miopen/execution_context.hpp

```diff
@@ -67,6 +67,7 @@ struct ExecutionContext
 {
     // Operation modes & environment
     bool do_search               = false;
+    bool db_update               = false;
```

Comment:
This is only initialized but not updated anywhere. Can we make this change when it's really needed (thus following the YAGNI principle)?

Reply:
It's used in fin, at conv_fin.hpp:559

Reply:
Yes, but the problem is that this PR does NOT include changes in conv_fin.hpp.

Reply:
This could be resolved by changes in #1588